modules/graphstudio/pages/explore-graph/actions.adoc

```diff
@@ -0,0 +1,38 @@
+= Actions
+
+The Actions tab in GraphStudio is a hub for actions that affect the entire graph database.
+
+== Clear all graph data
+
+image::clear-all-graph-data.png[]
+
+This command removes all loaded data from the schema while retaining all graph schemas and queries.
+
+Your user account must have the superuser role, and you must also be in Global View to clear all data from all graphs.
+You are required to type "clear all graph data" as a confirmation phrase to avoid accidentally performing this action.
+
+This operation will take about a minute depending on the size of your graphs and your hardware capability.
+
+[CAUTION]
+====
+Clear Graph Data deletes all data from your database irreversibly. Only the schema and queries will remain.
```

Comment:
This sounds a bit contradictory to me. Irreversibly makes it sound serious and "irreversible", but the next paragraph makes it sound kind of reversible. 

Are you saying that the data files you used to load data remain in your file system? That is different from saying data will remain in your system. The user could have data come in from everywhere, and simply the files they used for loading might not have all the changes that took place since they loaded them